internal/pkg/agent/application/secret/secret.go

```diff
@@ -2,6 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
+// Package secret manages application secrets.
```

Comment:
I touched this file so the linter would process it, previously there were failures that occurred in the linter: https://github.com/elastic/elastic-agent/pull/485/files